model/client.go

```diff
@@ -1108,7 +1108,7 @@ func (c *Client) LeaveGroup(installationID string, request *LeaveGroupRequest) e
 }
 
 // GetMultitenantDatabases fetches the list of multitenant databases from the configured provisioning server.
-func (c *Client) GetMultitenantDatabases(request *GetDatabasesRequest) ([]*MultitenantDatabase, error) {
+func (c *Client) GetMultitenantDatabases(request *GetMultitenantDatabasesRequest) ([]*MultitenantDatabase, error) {
```

Comment:
We should update paths used by this method to the new convention. That way updating the Provisioner client will solve the issue of migrating to the new API and no new integrations will use it.

Reply:
So, I intentionally didn't update the paths here, because this would make the client unusable against provisioners that don't have the new API yet. The plan was to later update these client methods and remove the old endpoints at the same time.

Reply:
Yeah that is true but only if you bump provisioner version in a `go.mod` and for a CLI against older provisioner which I guess would be the only issue.

Ideally, I think we should add new methods and deprecate those but as there are not many Provisioner API clients and we want to squeeze it in quickly I will leave it up to you.

Reply:
We chatted about this a bit offline and I feel this is a reasonable path forward for now. We can adjust the cleanup process a bit if needed.